hs/src/Reach/AST/SL.hs

```diff
@@ -180,7 +180,7 @@ primOpType DIV = [T_UInt, T_UInt] --> T_UInt
 primOpType MOD = [T_UInt, T_UInt] --> T_UInt
 primOpType PLT = [T_UInt, T_UInt] --> T_Bool
 primOpType PLE = [T_UInt, T_UInt] --> T_Bool
-primOpType PEQ = [T_UInt, T_UInt] --> T_Bool
+primOpType PEQ = T_Forall "a" ([T_Var "a", T_Var "a"] --> T_Bool)
```

Comment:
It is still used currently because I'm leveraging the `make_var` function for comparing dynamic int/digest/address values in `polyEq`. That function typechecks `PEQ`'s args